about/process/issue-tracking.rst

```diff
@@ -110,13 +113,14 @@ Bug Lifecycle
+    applicable for feature requests.
 
 - A *User* files a new bug using the issue-template.
-- The *QA-Team* adds the label **needs confirmation** and tries to work with the user to confirm the bug and add potentially missing information to the report. Once the report is complete a **team-label** will be added to the issue, the issue will be put on the **awaiting-triage-list** of the project and the label needs confirmation will be replaced with **bug**.
-- The affected *Team* will triage the issue and either reject (label **wontfix**, close and remove from the project) or accept the issue. The team decides if it they will fix the issue in-house (move to "Accepted" and assign a team member) or wait for a community developer to pick it up (Label **help wanted**, remove from the project board and provide hints on how to resolve the issue and further details on the how the fix should be implemented if necessary). For non-critical issues that are trivial to fix, the label **good first issue** can be added as well.
-- Once a *developer* is assigned and starts working on the issue, it is moved to "In Development". As soon as they have something to show for, the issue is closed and automatically moved to "Quality Assurance" for feedback from the QA team. If necessary, the developer will provide hints on how to test his patch in a comment on the issue.
-- The *QA-Team* tests the fix on all devices and provides feedback to the developer. If problems are found, the issue is re-opened and goes back to "Accepted", else it’s moved to "Release Candidate" to be included in the next release.
-- If not done already, the issue is added to the next milestone. Once the milestone is released, the issue is removed from the project board.
+- The *QA-Team* labels it **needs confirmation** and tries to work with the user to confirm the bug and add potentially missing info to the report. Once the report is complete a **team-label** is added to the issue, the issue will be put on the **awaiting-triage-list** of the project and the label needs confirmation will be replaced with **bug**.
+- The affected *Team* triages the issue and either rejects (label **wontfix**, closes and removes from the project) or accepts the issue. The team decides whether to fix the issue in-house (move to "Accepted" and assign a team member) or wait for a community developer to pick it up (by labeling it **help wanted**, removing it from the project board and providing hints on how to resolve the issue and further details on how the fix should be implemented if necessary). For non-critical issues trivial to fix, the label **good first issue** can be added as well.
+- Once a *developer* is assigned and starts working on the issue, it is moved to "In Development". As soon as there is something to show for, the issue is closed and automatically moved to "Quality Assurance" for feedback from the QA team.
+If necessary, the developer provides hints on how to test the patch in a comment on the issue.
```

Comment:
You insert a new line here. In general I am happy to see paragraphs of text broken into separate lines in the rst code. It makes it easier to see small changes in a diff if it's one sentence per line rather than a paragraph per line. 

Two points to this:
* here you do it inside an itemized list. Once you fixed the Jenkins build, please ensure that the itemized list still renders correctly and doesn't get broken by the newlines. I think it should be ok, but better safe than sorry
* provided that it IS ok to insert linebreaks, maybe you also want to do it in the other items in this list. you're touching every line already anyway, so review/diff already includes everything ... the best moment to take the hit and introduce the linebreaks

Reply:
Linebreaks are in. Build breaks not sure.